homeassistant/components/unifiprotect/event.py

```diff
@@ -57,10 +78,76 @@ def _async_update_device_from_protect(self, device: ProtectDeviceType) -> None:
         if (
             event
             and not self._event_already_ended(prev_event, prev_event_end)
-            and (event_types := description.event_types)
-            and (event_type := event.type) in event_types
+            and event.type == EventType.RING
+        ):
+            self._trigger_event(EventType.RING, {ATTR_EVENT_ID: event.id})
+            self.async_write_ha_state()
```

Comment:
This looks like it will break events other than RING.

apologizes if I posted this comment twice. I was having trouble with the connection.